deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-vertx/src/main/java/org/deeplearning4j/ui/api/UIServer.java

```diff
@@ -41,99 +38,81 @@
      * @return UI instance for this JVM
      * @throws RuntimeException if the instance has already started in a different mode (multi/single-session)
      */
-    public static synchronized UIServer getInstance() throws RuntimeException {
+    static UIServer getInstance() throws RuntimeException {
```

Comment:
Any reason we got rid of the public declaration?

Reply:
UIServer used to be an abstract class, now it's an interface (with java 8 default methods for consistency with the beta5 and earlier API). So it's public by definition.
Mainly because VertxUIServer class needs to extend AbstractVerticle for Vertx.
It's same API for users, they can still call UIServer.getInstance() as in beta5 and earlier.